apps/client/src/sttr_client/models/permit.js

```diff
@@ -1,3 +1,5 @@
+import { isNumber } from "util";
```

Comment:
Why use `isNumber` from `util` and not from `lodash.isnumber`, like you do in `models/question.js`? Is this different or maybe more lightweight?

Reply:
Good point! My IDE imported this function from `util` which I didn't even know existed. Anyway its been deprecated since node v4.0.0. I'll replace it with lodash's function.